packages/lexical-editor/src/components/Editor/RichTextEditor.tsx

```diff
@@ -75,21 +88,21 @@ const BaseRichTextEditor: React.FC<RichTextEditorProps> = ({
 
     return (
         <LexicalComposer initialConfig={initialConfig}>
-            <div ref={scrollRef}>
+            <div ref={scrollRef} style={{ ...editorSizeStyle() }}>
                 {/* data */}
                 <OnChangePlugin onChange={handleOnChange} />
                 {value && <LexicalUpdateStatePlugin value={value} />}
                 <ClearEditorPlugin />
                 {/* Events */}
                 {onBlur && <BlurEventPlugin onBlur={onBlur} />}
-                <AutoFocusPlugin />
+                {focus && <AutoFocusPlugin />}
                 {/* External plugins and components */}
                 {children}
                 <RichTextPlugin
                     contentEditable={
-                        <div className="editor-scroller">
-                            <div className="editor" ref={onRef}>
-                                <ContentEditable style={{ outline: 0 }} />
+                        <div className="editor-scroller" style={{ ...editorSizeStyle() }}>
+                            <div className="editor" ref={onRef} style={{ ...editorSizeStyle() }}>
+                                <ContentEditable style={{ outline: 0, ...editorSizeStyle() }} />
```

Comment:
Why call `editorSizeStyle()` 4 times, when you can call it once, assign to a const, and just pass that to all 4 elements?